### PR TITLE
Move the color picker to its own initializer file

### PR DESCRIPTION
**Pull request by Ridhwana**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
A small refactor that moves this code to it's own JS file so that it can be linted and compressed.

## Related Tickets & Documents
https://github.com/forem/forem/pull/10984/files#r509247051

## QA Instructions, Screenshots, Recordings
- Check that the color picker functions on http://localhost:3000/settings/profile still function correctly

### UI accessibility concerns?

_If your PR includes UI changes, please replace this line with details on how accessibility is impacted and tested. For more info, check out the [Forem Accessibility Docs](https://docs.forem.com/frontend/accessibility)._

## Added tests?

- [ ] Yes
- [x] No, because I jsut moved the js code from one file to another.
- [ ] I need help with writing tests

## Added to documentation?

- [ ] [Developer Docs](https://docs.forem.com) and/or [Admin Guide](https://forem.gitbook.io/forem-admin-guide/)
- [ ] README
- [x] No documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

File: app/assets/javascripts/initializePage.js [hunk 1]
```diff
@@ -65,6 +65,7 @@ function callInitializers() {
   initializeHeroBannerClose();
   initializeOnboardingTaskCard();
   initializeDateHelpers();
+  initializeColorPicker();
```
Comment (rhymes) [inline app/assets/javascripts/initializePage.js:68]: should the `initializeColorPicker` function exist in the `initializeColorPicker` file?

File: app/assets/javascripts/initializers/initializeColorPicker.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+var pickers = Array.from(document.getElementsByClassName('js-color-field'));
```
Comment (rhymes) [inline app/assets/javascripts/initializers/initializeColorPicker.js:1]: I think this whole file content needs to be wrapped inside:

```js
'use strict';

function initializeColorPicker() {
// ...
}

**Review by rhymes — CHANGES_REQUESTED** [review]
Thanks for the refactor, I think it's missing a bit

**Review by rhymes — APPROVED** [review]
It works now, thanks!

**Review by ludwiczakpawel — APPROVED** [review]

File: app/assets/javascripts/initializePage.js (hunk 1 above)
Comment (Ridhwana) [reply app/assets/javascripts/initializePage.js:68]: > It works now, thanks!

Thanks Rhymes, not sure how I missed that.

**Pull request merged**